peachjam/models/core_document_model.py

```diff
@@ -37,6 +38,29 @@
 from peachjam.storage import DynamicStorageFileField
 
 
+class Label(models.Model):
+    name = models.CharField(
+        _("name"), max_length=1024, unique=True, null=False, blank=False
+    )
+    code = models.SlugField(_("code"), max_length=1024, unique=True)
+    level = models.CharField(
+        _("level"), max_length=1024, null=False, blank=False, default="info"
```

Comment:
Add a `help_text="Level for styling, one of: primary, secondary, success, danger, warning or info."`